code/modules/materials/_materials.dm

```diff
@@ -306,6 +306,21 @@ INITIALIZE_IMMEDIATE(/obj/effect/gas_overlay)
 /decl/material/proc/products_need_process()
 	return (radioactivity>0) //todo
 
+// Returns the phase of the matterial at the given temperature and pressure
+// #FIXME: pressure is unused currently
+/decl/material/proc/phase_at_temperature(var/temperature, var/pressure = ONE_ATMOSPHERE)
+	//#TODO: implement plasma temperature and do pressure checks
+	if(temperature >= boiling_point)
+		return MAT_PHASE_GAS
+	else if(temperature >= heating_point)
+		return MAT_PHASE_LIQUID
+	else
```

Comment:
Extraneous `else`.

Reply:
Don't think it really changes anything if its there or not? The line will exec properly regardless?

Reply:
Yes, but it's an unnecessary line and indent as you return the immediate line above.

Reply:
done